storybook/README.md

```diff
@@ -1,17 +1,56 @@
-We are following the Component Story Format 3.
-Find more information about it here. https://storybook.js.org/blog/storybook-csf3-is-here
+# Goal
+Storybook is a tool for developing UI components in isolation. It makes building UIs organized and efficient.
+It also documents the components and their usage in an interactive way.
 
-If you are looking for a template from which to write stories, please do align with the ComposedChart.stories.tsx
+We want to migrate our current documentation to Storybook.
+The documentation is currently in the form of a static website, with embedded code sandboxes.
 
-- We want to use templates for rendering, and multiple stories with different parameters, content, data.
+## Structure
+As we want to migrate existing documentation to Storybook, we will want to reproduce the existing structure:
 
-- We want to document every single component in
--- the most simple manner
--- a complex usage
--- edge cases
+### API
+The API section contains stories for all components. The goal is to communicate a components behaviour as concise and simple as possible.
 
-- Additionally we will want to add stories that document complex usage of recharts with custom components.
+`storybook/stories/API/cartesian/Line.stories.tsx` can be considered a template for writing stories for the API section.
```

Comment:
We could link them to the file on GitHub here as well if we wanted (super minor suggestion definitely feel free to ignore):

```suggestion
[`storybook/stories/API/cartesian/Line.stories.tsx`](https://github.com/recharts/recharts/blob/master/storybook/stories/API/cartesian/Line.stories.tsx) can be considered a template for writing stories for the API section.
```

Reply:
Great idea. All suggestions are very welcome!